TimeTrackerApp/TimeTrackerApp/Utilites/Extensions/Color+Extension.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Color+Extension.swift
+//  TimeTrackerApp
+//
+//  Created by Sevda Gul Baran on 13.09.2022.
+//
+
+import Foundation
+import UIKit
+
+
+import UIKit
+
+extension UIColor {
+
+    class var pink: UIColor {
+        let const = 0xFD5B71
+        return UIColor.rgb(fromHex: const)
+    }
+    class var lightBackground: UIColor {
+        let const = 0xE5E5E5
+        return UIColor.rgb(fromHex: const)
+    }
+    class var lightGray: UIColor {
```

Comment:
variable öncesi ve sonrasında boşluk olması daha okunabilir yapıyor
```suggestion
    }
    
    class var lightGray: UIColor {
```